packages/runtime/src/helpers/functions.ts

```diff
@@ -364,58 +369,154 @@ const changeExtension = (file: string, extension: string) => {
   return join(dirname(file), base + extension)
 }
 
-const getSSRDependencies = async (publish: string): Promise<string[]> => {
+const getPrerenderManifest = async (publish: string) => {
```

Comment:
I extracted getting the prerenderManifest out of the prerender content functions so one these could be more testable potentially, but also, the job of those functions are more to iterated through the content of the prerendermanifest, not load it.